src/workers/newNotificationV2Mail.ts

```diff
@@ -109,15 +117,26 @@ export const notificationToTemplateId: Record<NotificationType, string> = {
   user_received_award: CioTransactionalMessageTemplateId.UserReceivedAward,
   organization_member_joined:
     CioTransactionalMessageTemplateId.OrganizationMemberJoined,
-  post_boost_completed: '79',
-  post_boost_first_milestone: '80',
   briefing_ready: '81',
   user_follow: '',
   marketing: '',
   new_user_welcome: '',
   announcements: '',
   in_app_purchases: '',
-  campaign_completed: '', // TODO: MI-1007 - wait for design's template id
+  campaign_completed: async (con, notification) => {
+    const campaign = await con
+      .getRepository(Campaign)
+      .findOneBy({ id: notification.referenceId });
```

Comment:
I don't like this, we add additional query which will fire when resolving, can we keep this map as is, you can split the campaigns in different notification types if they require different templates

Reply:
It is the worker. I don't think an additional query should hurt when fetching the template ID since we only fetch it once, regardless of the number of users to send to.

Reply:
Either way I would prefer not to add async complexity here, we have different type of notifications for this reason no?

Reply:
Is there is some other issue to having this not be same notification type that I am not aware of?